modules/cnf-topology-aware-lifecycle-manager-operator-update.adoc

```diff
@@ -47,7 +47,7 @@ spec:
           registryPoll:
             interval: 1h
 ----
-<1> The index image URL contains the desired Operator images. If the index images are always pushed to the same image name and tag, this change is not needed.
+<1> The index image URL contains the desired Operator images. If the index images are always pushed to the same image name and tag, this change is not needed. If you need to update the Operator image index, create a separate catalog source policy with the new image index or registry poll interval update. You must also prepare a separate subscription policy for the desired Operators to reference the new catalog source.
```

Comment:
It doesn't need to be a separate policy. In PGT, it can be another sourceFile with the same policy name but different CR name and spec. Similarly, subscriptions in PGT just need an update on its `source` field.